cou/utils/juju_utils.py

```diff
@@ -150,6 +150,14 @@ class Unit:
     machine: Machine
     workload_version: str
 
+    def __repr__(self) -> str:
```

Comment:
why `__repr__` instead of `__str__`?

Reply:
There is no specific reason. Generally `__repr__` is aimed at the programmer and `__str__` to the user. In this case I guess it's the same for both, so I think it's ok